modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -52,18 +52,30 @@ If you do not have access to Access Management, request access from your system
 
 == Generate a Certificate-Key Pair
 
-You’ll need a certificate-key pair for the internal load balancer in Runtime Fabric. If you have one already, both files must be of the .pem type, your keystore must have a passcode, and your Common Name should match the domain you’ll use in your Runtime Fabric. Otherwise, follow these steps to generate one for evaluation purposes.
+You’ll need a certificate-key pair for the internal load balancer in Runtime Fabric. If you have one already, both files must be of the .pem type, your keystore must have a passcode, and your Common Name should match the domain you’ll use in your Runtime Fabric. 
+
+RSA keys are the most common type of keys. RSA keys of 2K length offer the best compromise between security and performance. 
+[NOTE]
+RSA keys larger than 2K protect against brute force cracking and are appropriate for certificates that have expirations of many years. However, whenever key length is doubled, for example, from 2k to 4k, performance is reduced by a factor greater than 6.
+ 
+ECDSA keys are also supported. In most cases, ECDSA doubles the performance of a 2K RSA key. Supported curves are:
+
+* secp521r1 (P-521)
+* secp384r1 (P-384)
+* secp256r1 (also known as prime256v1 (P-256))
+
+Use the following steps to generate a certificate-key pair for evaluation purposes.
 
 . Run the following command on your machine to generate a certificate-key pair:
 +
 ----
-openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365
+openssl req -x509 -newkey rsa:2048 -keyout key.pem -out cert.pem -days 365
 ----
 
 . Type a passphrase for your key.
 . Fill out the requested information (Country, State, etc). When you’re asked for a Common Name, supply the domain you’ll use in your Runtime Fabric.
 
-If you use a wildcard for example, *.example.com in your Common Name, your application URLs use the following format: `{app-name}.example.com`. Otherwise, your application URLs use this format: `example.com/{app-name}`._
+If you use a wildcard for example, *.example.com in your Common Name, your application URLs use the following format: `{app-name}.example.com`. Otherwise, your application URLs use the format: `example.com/{app-name}`.
```

Comment:
```suggestion
If you use a wildcard (for example, *.example.com in your common name), your application URLs use the format`{app-name}.example.com`; otherwise, your application URLs use the format `example.com/{app-name}`.
```